runtime/doc/if_lua.txt

```diff
@@ -9,7 +9,31 @@ Lua Interface to Nvim					*lua* *Lua*
                                       Type <M-]> to see the table of contents.
 
 ==============================================================================
-1. Commands						*lua-commands*
+1. Importing modules					*lua-require*
+
+Neovim lua interface automatically adjusts `package.path` and `package.cpath` 
+according to effective &runtimepath value.  Adjustment happens after each time 
+'runtimepath' is changed, `package.path` and `package.cpath` are adjusted by 
+prepending directories from 'runtimepath' each suffixed by `/lua` and 
```

Comment:
@ZyX-I I am trying to understand this explanation. How is  `/lua` suffix involed in the following example? The example mentions `/foo` in &rtp,  and prepends `/foo` to some parts of package.cpath. But `/lua` is not involved.